app/controllers/idv/welcome_controller.rb

```diff
@@ -42,6 +42,11 @@ def analytics_arguments
       }.merge(ab_test_analytics_buckets)
     end
 
+    def confirm_welcome_step_available
+      # Don't allow access to the welcome step after the user has completed document capture.
+      redirect_to idv_phone_url if idv_session.pii_from_doc
```

Comment:
Ssn is the next step after document capture, so recommend redirecting there. You also need to check for idv_session.applicant, because it replaces idv_session.pii_from_doc when VerifyInfo is complete. We need the service/policy that @theabrad is working on!

```suggestion
      redirect_to idv_ssn_url if idv_session.pii_from_doc || idv_session.applicant
```